src/DotNet.Status.Web/.config/settings.json

```diff
@@ -136,6 +136,9 @@
     "ApiToken": "[vault(grafana-api-token)]",
     "TableUri": "[vault(deployment-table-sas-uri)]"
   },
+  "Annotations": {
+    "DeploymentsTableConnectionString": "[vault(deployment-table-sas-uri)]"
```

Comment:
Is there any universe in which this is different than Grafana.Annotations above? If not, can we not just use that setting to drive this behavior?

Reply:
I made them separate to keep responsibilities separate, and it is cheap.

Technically this controller only needs read-only access while the Grafana controller needs read-write. The Annotations controller also doesn't need the other secrets that the Grafana config brings.

Reply:
The responsibilities _aren't_ separate though. It's important they point to the same thing.  A better answer would be that there should be a service that encapsulates the table-ness of it all, and both controllers inject that instead.  But that's not cheap. I'd rather the annotations controller just inject an IOptions<GrafanaOptions>